xml/cap_depl_ldap.xml

```diff
@@ -0,0 +1,162 @@
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <link xlink:href="https://docs.cloudfoundry.org/uaa/uaa-user-management.html">The Cloud Foundry UAA command line interface (UAAC)</link>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     An LDAP server and the credentials for a user/service account with permissions to search the directory.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.cap.ldap-integration">
+  <title>Example LDAP Integration</title>
+  <para>
+   Run the following commands to complete the integration of your &cap; deployment and LDAP server.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Use UAAC to target your UAA server.
+    </para>
+<screen>&prompt.user;uaac target <replaceable>https://uaa.example.com:2793</replaceable></screen>
+   </listitem>
+   <listitem>
+    <para>
+     Authenticate to the UAA server as <literal>admin</literal> using the <literal>UAA_ADMIN_CLIENT_SECRET</literal> set in your <filename>scf-config-values.yaml</filename> file.
+    </para>
+<screen>&prompt.user;uaac token client get admin -s <replaceable>password</replaceable></screen>
+   </listitem>
+   <listitem>
+    <para>
+     Create the LDAP identity provider. A 201 response will be returned when the identity provider is successfully created. See the <link xlink:href="http://docs.cloudfoundry.org/api/uaa/version/4.21.0/index.html#ldap">UAA API Reference</link> and <link xlink:href="https://github.com/cloudfoundry/uaa/blob/master/docs/UAA-LDAP.md">&cf; UAA-LDAP Documentation</link>for information regarding the request parameters and additional options available to configure your identity provider.
+    </para>
+    <para>
+     The following is an example of a <literal>uaac curl</literal> command and its request parameters used to create an identity provider. Specify the parameters according to your LDAP server's credentials and directory structure. Ensure the user specifed in the <literal>bindUserDn</literal> has permissions to search the directory.
+    </para>
+<screen>&prompt.user;uaac curl /identity-providers?rawConfig=true -X POST \
+    -k \
+    -H 'Content-Type: application/json' \
+    -H 'X-Identity-Zone-Subdomain: scf' \
+    -d '{
+  "type" : "ldap",
+  "config" : {
+    "ldapProfileFile" : "ldap/ldap-search-and-bind.xml",
+    "baseUrl" : "<replaceable>ldap://ldap.example.com:389</replaceable>",
+    "bindUserDn" : "<replaceable>cn=admin,dc=example,dc=com</replaceable>",
+    "bindPassword" : "<replaceable>password</replaceable>",
+    "userSearchBase" : "<replaceable>dc=example,dc=com</replaceable>",
+    "userSearchFilter" : "<replaceable>uid</replaceable>={0}",
+    "ldapGroupFile" : "ldap/ldap-groups-map-to-scopes.xml",
+    "groupSearchBase" : "<replaceable>dc=example,dc=com</replaceable>",
+    "groupSearchFilter" : "<replaceable>member={0}</replaceable>"
+  },
+  "originKey" : "ldap",
+  "name" : "<replaceable>My LDAP Server"</replaceable>,
```

Comment:
Having the start quote be not replaceable, but the end quote being replaceable, seems kind of odd.